akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -204,6 +204,17 @@ object Source {
     })
 
   /**
+   * Create [[Source]] that will continually produce given elements in specified order.
+   *
+   * Start a new 'cycled' `Source` from the given elements. The producer stream of elements
+   * will continue infinitely by repeating the same sequence of elements provided during construction.
```

Comment:
Thanks for pointing that out! I did the update to both documentation and scaladoc.